src/common/components/search/SearchBar.tsx

```diff
@@ -126,6 +151,16 @@ const SearchBar: FC = () => {
                   <XIcon className="h-6 w-6 text-white hover:text-gray-400" />
                 </button>
               )}
+
+              {showKeybind && (
+                <div
```

Comment:
It's supposed to be more efficient to always create the component but programmatically hide it, e.g.

```
<div className={"... ${showKeybind ? '' : 'hidden'}"}...>
```